docs/language.rst

```diff
@@ -0,0 +1,170 @@
++------------------------+-------------------+-------------------+---------------------+
+| RANGE                  | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| GLOBAL                 | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| POINTER                | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| BBCOREPOINTER          | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| EXTERNAL               | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| THREADSAFE             | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| COMMENT                | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+|                                      SOLVE METHODs                                   |
++========================+===================+===================+=====================+
+| cnexp                  | yes               | yes               |                     |
++------------------------+-------------------+-------------------+---------------------+
+| euler                  | yes               | yes               |                     |
++------------------------+-------------------+-------------------+---------------------+
+| derivimplicit          | yes               | yes               |                     |
++------------------------+-------------------+-------------------+---------------------+
+| sparse                 | yes               | yes               |                     |
++------------------------+-------------------+-------------------+---------------------+
+| runge                  | yes               | no                |                     |
++------------------------+-------------------+-------------------+---------------------+
+| after_cvode            | yes               | no                |                     |
++------------------------+-------------------+-------------------+---------------------+
+| cvode_t                | yes               | no                |                     |
++------------------------+-------------------+-------------------+---------------------+
+| cvode_v                | yes               | no                |                     |
++------------------------+-------------------+-------------------+---------------------+
```

Comment:
there are some additional methods here, see `lookup` or `method->name` in [here](https://github.com/BlueBrain/mod2c/blob/788af0ef16821bb51e9b80b5140f5d90a780070a/src/mod2c_core/solve.c#L166).

In my presentation I had following:

```
adams, runge, euler, adeuler, heun, adrunge, gear, newton, simplex, simeq, seidel, 
_advance, sparse, derivimplicit, cnexp, clsoda, after_cvode 
```